### PR TITLE
Add AWS resources

### LINKED ISSUE #272: Missing AWS smaller resources in storage
There are some of the smaller icons that are missing in storage such as 
* bucket
* bucket with objcet
* snapshot
* filesystem

These can are available from the artchitecture download: https://aws.amazon.com/architecture/icons/

I can create a PR for this.

### PR DESCRIPTION
**Pull request by semnil**
I'm using the results of this wonderful project a lot. Thank you. 😄 

Add AWS resources (from the [AWS architecture icons](https://aws.amazon.com/architecture/icons/)).
I may have added too many at once. 😅 
If the file names aren't right, I'll fix it!

Closes #272

### CODE REVIEW

File: docs/nodes/aws.md [hunk 1]
```diff
@@ -272,17 +420,25 @@ Node classes list of aws provider.
...
 - **diagrams.aws.network.CloudMap**
+- **diagrams.aws.network.CloudFrontDownloadDistribution**
+- **diagrams.aws.network.CloudFrontEdgeLocation**
+- **diagrams.aws.network.CloudFrontStreamingDistribution**
 - **diagrams.aws.network.CloudFront**, **CF** (alias)
 - **diagrams.aws.network.DirectConnect**
 - **diagrams.aws.network.ElasticLoadBalancing**, **ELB** (alias)
+- **diagrams.aws.network.ElbApplicationLoadBalancer**
+- **diagrams.aws.network.ElbClassicLoadBalancer**
+- **diagrams.aws.network.ElbNetworkLoadBalancer**
```
Comment (semnil) [inline docs/nodes/aws.md:441]: Perhaps removing the "Elb" prefix might be more user-friendly. 🤔

**Comment by wolfspyre** [comment]
This adds a lot of value.. @mingrammer - Thoughts?

**Review by mingrammer — COMMENTED** [review]
Looks awesome! I have some suggestions.

It would be good to add aliases for some nodes.

`AMI` for `EC2Ami`
`ALB` for `ElbApplicationLoadBalancer`
`ELB` for `ElbClassLoadBalancer`
`NLB` for `ElbNetworkLoadBalancer`

File: docs/nodes/aws.md (hunk 1 above)
Comment (mingrammer) [reply docs/nodes/aws.md:441]: How about alias these nodes?

**Comment by mingrammer** [comment]
@semnil @wolfspyre  Sorry for the late review :( I left a review comment.

**Comment by semnil** [comment]
@mingrammer Thank you for your review!
I added the suggested aliases. 😄

**Review by mingrammer — APPROVED** [review]
LGTM! Thank you.

**Pull request merged**